posthog/clickhouse/migrations/0148_sessions_v3.py

```diff
@@ -0,0 +1,16 @@
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.models.raw_sessions.sql_v3 import (
+    DISTRIBUTED_RAW_SESSIONS_TABLE_SQL_V3,
+    RAW_SESSIONS_CREATE_OR_REPLACE_VIEW_SQL_V3,
+    RAW_SESSIONS_TABLE_SQL_V3,
+    WRITABLE_RAW_SESSIONS_TABLE_SQL_V3,
+)
+
+operations = [
+    run_sql_with_exceptions(WRITABLE_RAW_SESSIONS_TABLE_SQL_V3()),
```

Comment:
@PostHog/clickhouse would you mind checking this SQL too? and role / on cluster clauses, etc